samples/tab-sso/nodejs/src/client/scripts/ssoDemo.js

```diff
@@ -18,29 +16,23 @@
                     reject("Error getting token: " + error);
                 }
             });
-
         });
-
     }
 
     // 2. Exchange that token for a token with the required permissions
     //    using the web service (see /auth/token handler in app.js)
     function getServerSideToken(clientSideToken) {
 
-        display("2. Exchange for server-side token");
-
         return new Promise((resolve, reject) => {
-
             microsoftTeams.getContext((context) => {
-
-                fetch('/auth/token', {
+                fetch('/getProfile', {
```

Comment:
Why getProfile why not acquireTokenOnBehalfOf ?

Reply:
renamed it.